Extract api_surface configurability (#21953)
dotnet/docs: docs/fundamentals/code-analysis/quality-rules/ca5382.md

## Configure code to analyze

Use the following options to configure which parts of your codebase to run this rule on.

- [Exclude specific symbols](#exclude-specific-symbols)
- [Exclude specific types and their derived types](#exclude-specific-types-and-their-derived-types)

> This rule has the following configurable options that can be configured for just this rule, for all rules, or for all rules in this category (Security). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).

Reviewer: ```suggestion
You can configure these options for just this rule, for all rules, or for all rules in this category (Security). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
```
